core/indigo-core/molecule/base_molecule.h

```diff
@@ -99,33 +99,50 @@ namespace indigo
     class Molecule;
     class QueryMolecule;
 
-    class MetaObjectsInterface
+    class DLLEXPORT MetaData
```

Comment:
Let's move this class to standalone file, and also let's rename it, Metadata is too generic. At least to BaseMoleculeMetadata, if you don't have better name.

Reply:
renamed to MetaDataStorage and moved to metadata_storage.h and metadata_storage.cpp